test/k8s-integration/main.go

```diff
@@ -111,6 +112,12 @@ func main() {
 
 	if !*bringupCluster {
 		ensureVariable(kubeFeatureGates, false, "kube-feature-gates set but not bringing up new cluster")
+	} else {
+		ensureVariable(imageType, true, "image type is a required flag. Available options include 'cos' and 'ubuntu'")
+	}
+
+	if *platform == "windows" && *inProw {
+		ensureFlag(bringupCluster, false, "bringupCluster is set to false if it is for prow test in windows cluster")
```

Comment:
Actually bringupCluster is not supported for any windows testing right? (even outside of prow)

Reply:
right now yes. Later we could add an option to bring up windows cluster.
Remove the inPros check.